internal/repositories/devicedefinition/devicedefinition.go

```diff
@@ -254,7 +262,7 @@ func (r *Repository) GetDeviceDefinitions(ctx context.Context, tableID, first *i
 	nodes := make([]*gmodel.DeviceDefinition, len(all))
 
 	for i, dv := range all {
-		gv, err := ToAPI(&dv)
```

Comment:
So it won't show up in the list view? I know it's not terribly useful in that case because the manufacturer is the parent, but it seems inconsistent, I don't like it.

Reply:
sounds good will add to the list too